### DISCUSSION TITLE
Documentation, re "twice as fast (100% faster)"

### DISCUSSION THREAD

**Discussion by morrismichaelc** [mdx-js/mdx#2066] [General] [answered]
This is a pet peeve of mine; I certainly don't want to file a bug report, but **I think the home page should be corrected** regarding MDX 2 features.

It says:

> 🚴 Generated code runs twice as fast (100% faster)

* But "twice as fast" is actually a nonsensical phrase; it must be 2x something, by 2x what? The time it currently takes to run? That's definitely wrong...
* Also, 100% faster implies it runs with a 100% reduction in elapsed time, which would mean instantaneously.

You pretty clearly mean it runs in half the time; this is 50% faster.a

**Comment by wooorm** [comment, marked as answer]
- I believe the phrase to be sensical, because people understand it.
- This block is marketing, about an new release, that should get folks excited, it doesn’t have to adhere to scientific levels of correctness
- As you say, it is pretty clear

**Comment by morrismichaelc** [comment]
Thanks for reading & considering the comment! 

On 2022-06-24 00:36, Titus wrote:

> * I believe the phrase to be sensical, because people understand it.
> * This block is marketing, about an new release, that should get folks excited, it doesn't have to adhere to scientific levels of correctness
> * As you say, it is pretty clear
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe [2].
> You are receiving this because you authored the thread.Message ID: ***@***.***>
 

Links:
------
[1]
https://github.com/orgs/mdx-js/discussions/2066#discussioncomment-3015727
[2]
